warehouse/templates/manage/manage_base.html

```diff
@@ -102,6 +101,11 @@ <h2 class="sub-title">{% trans %}Two factor authentication (2FA){% endtrans %}</
 
       </tbody>
     </table>
+    {% if user.has_single_2fa %}
+    <div class="callout-block">
+        <p>{% trans %}To remove this 2FA method, you must first add another method.{% endtrans %}</p>
+    </div>
+    {% endif %}
```

Comment:
I think I'd prefer to move this up under the horizontal line, but before the table of two factor methods.

Reply:
I've made this change. I'm not great with CSS, so I hope my approach isn't too hacky. I've updated the pic in the PR desc.